packages/financial-templates-lib/src/logger/Logger.ts

```diff
@@ -74,10 +81,8 @@ function createNewLogger(injectedTransports = [], transportsConfig = {}, botIden
       winston.format.json()
     ),
     transports: [...createTransports(transportsConfig), ...injectedTransports],
-    exitOnError: process.env.EXIT_ON_ERROR ? process.env.EXIT_ON_ERROR : false,
+    exitOnError: !!process.env.EXIT_ON_ERROR,
```

Comment:
this is new syntax to me! pretty cool: _"Converts Object to boolean. If it was falsey (e.g. 0, null, undefined, etc.), it will be false, otherwise, true.

"_

Reply:
Yeah, this doesn't matter in js since things are often used as booleans implicitly, but typescript really cares about stuff like this. If you call it a bool it needs to be a bool.